src/rules/converters/import-blacklist.ts

```diff
@@ -0,0 +1,52 @@
+import { RuleConverter } from "../converter";
+import { RequireAtLeastOne } from "../../utils";
+
+type ESLintOptionPath = {
+    name: string;
+    importNames: string[];
+    message?: string;
+};
+type ESLintSimpleOption = string[];
+type ESLintComplexOption = RequireAtLeastOne<{
+    paths: (string | ESLintOptionPath)[];
+    patterns: string[];
+}>;
+type ESLintOptions = ESLintSimpleOption | ESLintComplexOption;
+
+export const convertImportBlacklist: RuleConverter = tslintRule => {
+    const tslintRules = tslintRule.ruleArguments;
+    let ruleArguments: ESLintOptions = [];
+
+    if (tslintRules.every(isString)) {
+        ruleArguments = tslintRules;
+    } else {
+        ruleArguments = [
+            tslintRules.reduce((rules, rule) => {
+                if (!Array.isArray(rule)) {
+                    const eslintRule = isString(rule)
+                        ? rule
+                        : {
+                              name: Object.keys(rule)[0],
+                              importNames: Object.values(rule)[0] as string[],
+                          };
+                    return { ...rules, paths: [...(rules.paths || []), eslintRule] };
+                } else if (Array.isArray(rule)) {
+                    return { ...rules, patterns: [...(rules.patterns || []), ...rule] };
```

Comment:
ESLint and TSLint use different strategies to match patterns.

* ESLint uses [`ignore`](https://github.com/eslint/eslint/blob/dd1c30e35f05ed332e2abbd3d4d53635efde74b8/lib/rules/no-restricted-imports.js#L11)
* TSLint uses [regular expressions](https://palantir.github.io/tslint/rules/import-blacklist) 

There's nothing we can reasonably do to convert from one form to the other 😢 so the best way to deal with the difference is to add a `notices` entry mentioning the format is different.